plugin.json

```diff
@@ -2,7 +2,7 @@
     "id": "com.mattermost.aws-sns",
     "name": "AWS SNS",
     "description": "Send alert notifications from Amazon AWS CloudWatch to Mattermost channels via AWS SNS.",
-    "homepage_url": "https://github.com/mattermost/mattermost-plugin-aws-SNS",
+    "homepage_url": "https://mattermost.com/pl/mattermost-plugin-aws-SNS",
```

Comment:
Not sure if we need to change this. The original URL is not a gitbook URL

Reply:
Personally I'm thinking this just causes confusion if we change from pointing to the repo to somewhere else. I was under the impression the link would take me to the Mattermost docs site, but it's just a redirect to the repo's readme. As mentioned in the spreadsheet linked in the PR description:

> In Yoast because of hash - but since it's an external link doesn't really matter

It seems that it "doesn't matter" if we make this change or not, so I propose we keep it how it is to avoid confusion

Reply:
Same for the following PRs:

- https://github.com/mattermost/mattermost-plugin-circleci/pull/94
- https://github.com/mattermost/mattermost-plugin-gitlab/pull/415
- https://github.com/mattermost/mattermost-plugin-jira/pull/998

Is the plan to relocate some of the information in the readmes to another location?

Reply:
Long term, yes. Mattermost-supported plugin README content will likely move back to the product docs site, though a timeline for that work hasn't been set yet.

These changes were applied to bring Mattermost-supported content in line with our [Handbook policy](https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#how-do-i-create-redirects-for-in-product-documentation) of linking to documentation via permalinks managed by Marketing. 

If this change will introduce confusion in the short term, I'm 0/5 on keeping these proposed changes and open to reverting to the GitHub links instead. We could decide to address links if/when we decide to move content in the future.

Reply:
> If this change will introduce confusion in the short term, I'm 0/5 on keeping these proposed changes and open to reverting to the GitHub links instead. We could decide to address links if/when we decide to move content in the future.

Yes I would prefer this for now. @hanzei Thoughts on this?

Reply:
Sounds good. Thank you both for your feedback and input. I'll close this PR and the other PRs introducing the same updates, as @mickmister listed above.